### PR TITLE
Add maintenance and host operation

### PR DESCRIPTION
**Pull request by akitsul**
Add maintenance search, create, remove.
Add host search and remove.

### CODE REVIEW

File: docs.go [hunk 1]
```diff
@@ -119,6 +121,40 @@ Workflow options:
...
+      Remove specified <maintenance>.
+
+    -z --read-stdin
+      Read hosts from stdin.
+
+  -H --hosts
+    Search and operate with hosts.
+
+    -r --remove
+      Remove hosts.
```
Comment (tears-of-noobs) [inline docs.go]: This description doesn't so clear to the understanding. It's a boolean argument?

File: handle_hosts.go [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+	return nil
+}
+
+func handleRemoveHosts(
+	zabbix *Zabbix,
+	config *Config,
+	args map[string]interface{},
+) error {
+	var (
+		removeHost, _ = args["--remove"].(string)
```
Comment (tears-of-noobs) [inline handle_hosts.go:68]: What is `removeHost`? It's a pattern or a single hostname?

File: docs.go (hunk 1 above)
Comment (akitsul) [reply docs.go]: It's string, fixed description.

File: handle_hosts.go (hunk 2 above)
Comment (akitsul) [reply handle_hosts.go:68]: It's single hostname, this is checked after search host and return error https://github.com/zarplata/zabbixctl/blob/maintenance%2Bhost%2Btest/handle_hosts.go#L81

File: group.go [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package main
+
+type Group struct {
```
Comment (tears-of-noobs) [inline group.go:3]: Don't capitalize the name of structs, constants etc. if it not really necessary. In this case, you have a package `main` only which won't be imported, so you don't need any exported struct/variable.

**Pull request merged**